agent/internal/client/base_client.go

```diff
@@ -759,10 +757,6 @@ func (c *baseCommunicator) GetDistroByName(ctx context.Context, id string) (*res
 
 // StartTask marks the task as started, and sends traceId and diskDevices to be stored with the task.
 func (c *baseCommunicator) StartTask(ctx context.Context, taskData TaskData, traceID string, diskDevices []string) error {
-	grip.Info(message.Fields{
-		"message": "started StartTask",
-		"task_id": taskData.ID,
-	})
```

Comment:
This and the one below don't seem related to this change so just double checking that you meant to remove it. (I don't think we need it so removing seems fine)